uefi/flashmastersection.go

```diff
@@ -9,23 +9,24 @@ import (
 // FlashMasterSectionSize is the size in bytes of the FlashMaster section
 const FlashMasterSectionSize = 12
 
+// RegionPermissions holds the
```

Comment:
holds the what? the what? We don't have much time here, I've got to know!

Reply:
I don't know why I'm dropping all these comments dammit.